lib/chef/provider/mount.rb

```diff
@@ -176,7 +176,12 @@ def wait_until_unmounted(tries)
       # Returns the new_resource device as per device_type
       def device_fstab
         # Removed "/" from the end of str, because it was causing idempotency issue.
```

Comment:
Can you expand this comment a bit since we're beyond the single chomp scenario described there